keras/wrappers/scikit_learn.py

```diff
@@ -115,23 +151,17 @@ def fit(self, X, y):
         history : object
             Returns details about the training history at each epoch.
         """
-        if len(y.shape) == 1:
-            self.classes_ = list(np.unique(y))
-            if self.loss == 'categorical_crossentropy':
-                y = to_categorical(y)
-        else:
-            self.classes_ = np.arange(0, y.shape[1])
 
-        self.compiled_model_ = copy.deepcopy(self.model)
+        self.compiled_model_ = copy.deepcopy(self.model.model)
```

Comment:
In general private attributes should prefixed with `_` rather than the opposite.